build.cmd

```diff
@@ -188,7 +152,7 @@ echo "Building Managed code ... "
 echo "#################################"
 set _dotnet=%_dotnetRoot%\dotnet.exe
 
-if "%Configuration:~-5%" == "Py3.7" set VerifyManifest=True
+if "%Configuration:~-5%" == "Py3.6" set VerifyManifest=True
```

Comment:
Why was this version lowered vs. increased since the default configuration is now Python 3.8?

Reply:
this is non issue. it should work both on 3.7 or 3.6

---
In reply to: [441269724](https://github.com/microsoft/NimbusML/pull/494#discussion_r441269724) [](ancestors = 441269724)